JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1 +1,13 @@
-The **DataGrid** widget allows you to&nbsp;customize its toolbar using the **onToolbarPreparing** function. You can add new items to&nbsp;the toolbar or&nbsp;customize the existing ones. In&nbsp;this demo, the toolbar contains a&nbsp;collection of&nbsp;other widgets from the DevExtreme HTML5&nbsp;UI Widgets library.
\ No newline at end of file
+Toolbar provides additional control elements. To add and customize the toolbar, implement the onToolbarPreparing option. You can configure an existent element or add a DevExtreme or custom widget in this function. For this, manipulate the toolbarOptions.items array located in the parameter object.
```

Comment:
It is clear that a toolbar contains controls and that the DE Toolbar widget implements the toolbar functionality. Keeping this in mid, I used a single paragraph instead of two, made some changes and used the suggested options. '*Additional* controls' is used to highlight that the widget works properly without them. The paragraph is as follows:

The Toolbar widget contains additional controls for DevExtreme widgets. To add and customize DataGrid's Toolbar, implement the onToolbarPreparing function. Its parameter's toolbarOptions field contains the [configuration]() object. Modify the the **toolbarOptions**.[items] array to manipulate and add elements to the Toolbar. They can be DevExtreme widgets and custom elements.